### PR TITLE
Tests: Add unittest for da config

### PR DESCRIPTION
**Pull request by pwirth**

### CODE REVIEW

File: tests/phpunit/integration/DataAccountingConfigTest.php [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+	}
+
+	/**
+	 * @covers \DataAccounting\Config\DataAccountingConfig::get
+	 */
+	public function testExtendMWConfig() {
+		$config = $this->getServiceContainer()->getMainConfig();
+		$daConfig = $this->getServiceContainer()->getConfigFactory()->makeConfig( 'da' );
+		foreach ( $this->configData as $key => $val ) {
+			$this->setMwGlobals( "wg$key", $val );
```
Comment (rht) [inline tests/phpunit/integration/DataAccountingConfigTest.php:67]: Isn't this supposed to be "da$key"?

Comment (pwirth) [reply tests/phpunit/integration/DataAccountingConfigTest.php:67]: "wg$key" is correct here, as this case tests that keys, that are in the mainconfig are available in the da config as well

Comment (rht) [reply tests/phpunit/integration/DataAccountingConfigTest.php:67]: Should "da$key" namespace be tested as well?

Comment (pwirth) [reply tests/phpunit/integration/DataAccountingConfigTest.php:67]: oh, do you mean setting globals with prefix "da" should be tested to appear in the config then?
sure, i will add that

Comment (rht) [reply tests/phpunit/integration/DataAccountingConfigTest.php:67]: Yeah exactly.

Comment (pwirth) [reply tests/phpunit/integration/DataAccountingConfigTest.php:84]: done. i also added a testcase for the config handlers set method

**Comment by rht** [comment]
Merged!

**Pull request merged**